website/src/views/hooks/useMediaQuery.test.tsx

```diff
@@ -1,36 +1,44 @@
 import { act, render } from '@testing-library/react';
 import { breakpointUp } from 'utils/css';
-import { FC } from 'react';
-import { mockWindowMatchMedia } from 'test-utils/mockDom';
+import type { FC } from 'react';
+import { mockDom, mockDomReset, mockWindowMatchMedia } from 'test-utils/mockDom';
 import useMediaQuery from './useMediaQuery';
 
 const Tester: FC = () => {
   const matchedBreakpoint = useMediaQuery(breakpointUp('md'));
-
-  if (matchedBreakpoint) {
-    return <div />;
-  }
-  return null;
+  return matchedBreakpoint ? <>matched</> : <>no match</>;
 };
 
 describe(useMediaQuery, () => {
-  function make() {
-    return render(<Tester />);
-  }
+  beforeEach(() => {
+    mockDom();
+  });
+
+  afterEach(() => {
+    mockDomReset();
+  });
 
-  test('should hide module when screen size is small', () => {
+  test('should display correct module based on matching', () => {
```

Comment:
Hmm the "module" stuff was specific to what GlobalSearchContainer was doing.

```suggestion
  test('should return whether media queries were matched', () => {
```

This also means that a more extensive test would expect that `window.matchMedia` was called with the correct media queries passed into `useMediaQuery` ("correct" because `useMediaQuery` transforms the input queries, so they're not simply passed through directly to `window.matchMedia`)